makedoc.g

```diff
@@ -26,7 +25,11 @@ UrlEntity := function(name, url)
 end;
 
 PackageEntity := function(name)
-  return UrlEntity(name, PackageInfo(name)[1].PackageWWWHome);
+  if LoadPackage(name, false) <> fail then
```

Comment:
Minor suggestion, you could just check whether the package is (available to be) loaded, rather than trying to actually load it – this might have a marginally higher success rate:
```suggestion
  if TestPackageAvailability(name) <> fail then
```

Reply:
Good idea, I had `IsPackageMarkedForLoading` before, which was not quite right, but didn't remember/know about `TestPackageAvailability`.